ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -217,6 +224,38 @@ void ompl_interface::ModelBasedPlanningContext::useConfig()
   if (cfg.empty())
     return;
 
+
+  ompl::base::OptimizationObjectivePtr objective;
+  it = cfg.find("optimization_objective");
+  if (it == cfg.end())
+  {
+    logInform("%s: No optimization objective specified, defaulting to PathLengthOptimizationObjective");
+    objective.reset(new ompl::base::PathLengthOptimizationObjective(ompl_simple_setup_->getSpaceInformation()));
+  }
```

Comment:
could you rewrite the default case to something like this:
```
std::string objective;
it = cfg.find("optimization_objective");
if (it == cfg.end())
{
   objective = "PathLengthOptimizationObjective";
   logInform(" ... defaulting to " + objective); // with appropriate string type conversions I suppose...
}
else
{
  objective = it->second;
}
```

This avoids redundancy and the big else-block afterwards.